www/scripts/codecheckerviewer/ListOfRuns.js

```diff
@@ -36,6 +36,13 @@ function (declare, domConstruct, ItemFileWriteStore, topic, Dialog, Button,
     return prettyDuration;
   }
 
+  function prettifyStatus(statusCounts) {
+    items = []
```

Comment:
I think it would be better to use the array map function here:
```javascript
return statusCounts.map(function (current, index, arr) {
  return index + '&nbsp(' + current + ')';
}).join(', ');
```